### PR TITLE
Add digital ocaen

### PR DESCRIPTION
**Pull request by vzholudev**

### CODE REVIEW

File: digitalocean/terraform/backend.tf [hunk 1]
```diff
@@ -0,0 +1,15 @@
+terraform {
+  backend "s3" {
```
Comment (bashofmann) [inline digitalocean/terraform/backend.tf]: The examples should also easily be runnable by external customers. Therefor remove the S3 backend here. We should only use the default file backend for the state.

File: digitalocean/terraform/main.tf [hunk 2]
```diff
@@ -0,0 +1,6 @@
+module "test-cluster" {
```
Comment (bashofmann) [inline digitalocean/terraform/main.tf]: I don't think a module is necessary here, I think I'd prefer a flat structure here.

File: digitalocean/terraform/modules/cluster-setup/README.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
+# Custer and network setup
+This module is for baisic setup VPC and k8s cluster in DigitalOcean project.
```
Comment (bashofmann) [inline digitalocean/terraform/modules/cluster-setup/README.md]:
```suggestion
This module is for a basic VPC and k8s cluster setup in a DigitalOcean project. 
```

File: digitalocean/README.md [hunk 4]
```diff
@@ -0,0 +1 @@
+# Qdrant Hybrid Cloud Example for DigitalOcean
```
Comment (bashofmann) [inline digitalocean/README.md]: This readme needs some basic info on how to use this.

**Review by bashofmann — CHANGES_REQUESTED** [review]
Sorry for the delay, I was out-of-office for a couple days. The following points need to be addressed:

* The vpc `ip_range` needs to be configurable through a variable
* The provider configuration is missing, something like
```
variable "do_token" {}

provider "digitalocean" {
  token = var.do_token
}
```
* Add `*.tfvars` to `.gitignore`
* Add a `terraform.tfvars.example` file, see https://github.com/rancher/quickstart/blob/master/rancher/do/terraform.tfvars.example
* Auto generate this tfvars example file and the docs, see https://github.com/rancher/quickstart/blob/master/Makefile#L15-L21
* Write the generated kubeconfig (https://registry.terraform.io/providers/digitalocean/digitalocean/latest/docs/resources/kubernetes_cluster#raw_config) to a ignored file for easy usage

**Comment by vzholudev** [comment]
@bashofmann please take a look one more time

**Review by bashofmann — APPROVED** [review]

**Pull request merged**